### PR TITLE
MGMT-19058: Add OpenShif AI operator

### PR DESCRIPTION
**Pull request by jhernand**
This patch adds to the operators page of the cluster creation wizard a new check-box that will be used to enable the installation of the OpenShift AI operator.

https://github.com/user-attachments/assets/e0cd543b-89fe-4e3e-8ade-813dda350dfb

Related: https://issues.redhat.com/browse/MGMT-19058
Related: https://issues.redhat.com/browse/MGMT-19057

### CODE REVIEW

File: libs/ui-lib/lib/cim/components/ClusterDeployment/_wizardTransition.test.ts
Comment (ammont82) [inline libs/ui-lib/lib/cim/components/ClusterDeployment/_wizardTransition.test.ts:1]: Better don't touch the CIM side

File: libs/ui-lib/lib/cim/components/ClusterDeployment/wizardTransition.ts
[the remark this answers is not in the document]
Comment (jhernand) [reply libs/ui-lib/lib/cim/components/ClusterDeployment/wizardTransition.ts:1]: Done.

File: libs/ui-lib/lib/cim/components/ClusterDeployment/_wizardTransition.test.ts
Comment (jhernand) [reply libs/ui-lib/lib/cim/components/ClusterDeployment/_wizardTransition.test.ts:1]: Done.

**Review by ammont82 — DISMISSED** [review]

**Comment by jhernand** [comment]
Added a commit to improve the display of requirements and dependencies:

**Pull request merged**